.github/workflows/main.yaml

```diff
@@ -1,36 +1,261 @@
+            start: 4
+          - size: 7680
+            split: 20
+            start: 6
+          - size: 7680
+            split: 20
+            start: 8
+          - size: 7680
+            split: 20
+            start: 10
+          - size: 7680
+            split: 20
+            start: 12
+          - size: 7680
+            split: 20
+            start: 14
+          - size: 7680
+            split: 20
+            start: 16
+          - size: 7680
+            split: 20
+            start: 18
+
+          - size: 8192
+            split: 24
+            start: 0
+          - size: 8192
+            split: 24
+            start: 2
+          - size: 8192
+            split: 24
+            start: 4
+          - size: 8192
+            split: 24
+            start: 6
+          - size: 8192
+            split: 24
+            start: 8
+          - size: 8192
+            split: 24
+            start: 10
+          - size: 8192
+            split: 24
+            start: 12
+          - size: 8192
+            split: 24
+            start: 14
+          - size: 8192
+            split: 24
+            start: 16
+          - size: 8192
+            split: 24
+            start: 18
+          - size: 8192
+            split: 24
+            start: 20
+          - size: 8192
+            split: 24
+            start: 22
+
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/download-artifact@v2
+      with:
+        name: candidate-${{ matrix.size }}
 
     - name: Validate
       run: |
-        ssh-keygen -f candidate-${{ matrix.size }} -T validated-${{ matrix.size }}
+        # Prepare all input as shell variables.
+        size=${{ matrix.size }}
+        first_index=${{ matrix.start }}
+        second_index=$(( $first_index + 1 ))
+        first_index=`printf %02d $first_index`
+        second_index=`printf %02d $second_index`
+        # Some debugging.
+        ls -al *
+        declare -p
+
+        # The action.
+        ssh-keygen -M screen -f g-$size.$first_index  v-$size-$first_index &
+        PID1=$!
+        ssh-keygen -M screen -f g-$size.$second_index  v-$size-$second_index &
+        PID2=$!
+
+        FAIL=0
+        wait $PID1 || let "FAIL+=1"
+        wait $PID2 || let "FAIL+=1"
+        exit $FAIL
+
+    - uses: actions/upload-artifact@v2
+      with:
+        name: validated
+        path: v-*
+        retention-days: 1
+
+
+  combine:
+    needs: screen
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v2
+    - uses: actions/download-artifact@v2
+      with:
+        name: validated
+
+    - name: Debug
+      run: |
+        ls -al *
+        wc -l v-*
+
+    - name: OpenSSH /etc/ssh/moduli
+      run: |
+        now=`date -Iminutes`
+        echo "# https://github.com/chevah/ssh-moduli/ $now" > etc_ssh_moduli
+        echo "# Time Type Tests Tries Size Generator Modulus" >> etc_ssh_moduli
+
+        cat v-* | sort -u >> etc_ssh_moduli
+
+    - name: Chevah Python Dict
```

Comment:
I think that this can be removed. nobody should load this data as python code

Instead it should be a data file and a separate code to load and parse the data.